### PR TITLE
Fixing cls issue

### PR DESCRIPTION
**Pull request by wfulp**
After testing old reports run fine with the fix

### CODE REVIEW

**Comment by monicagerber** [comment]
Do you have documentation of the issue somewhere? I thought we fixed this with https://github.com/FredHutch/VISCtemplates/pull/44?

**Comment by wfulp** [comment]
Here is some documentation of the issue. Since we couldn't address the issue the same way `rmarkdown` did I copied from their resulting .tex file (using the default rmarkdown template).

https://github.com/rstudio/rmarkdown/issues/1649

File: inst/rmarkdown/templates/visc_report/resources/template.tex [hunk 1]
```diff
@@ -121,15 +121,26 @@
...
 
-% pandoc now requires a cslreferences environment to be defined in the latex template
-$if(csl-refs)$
 \newlength{\cslhangindent}
 \setlength{\cslhangindent}{1.5em}
-\newenvironment{cslreferences}%
-  {$if(csl-hanging-indent)$\setlength{\parindent}{0pt}%
-  \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces$endif$}%
-  {\par}
-$endif$
```
Comment (monicagerber) [inline inst/rmarkdown/templates/visc_report/resources/template.tex:132]: Referencing this document: https://github.com/jgm/pandoc/blob/master/data/templates/default.latex

Looks like you need to add back in `$if(csl-refs)$` and `$endif$`. Then I will merge!

**Review by monicagerber — CHANGES_REQUESTED** [review]

File: inst/rmarkdown/templates/visc_report/resources/template.tex (hunk 1 above)
Comment (wfulp) [reply inst/rmarkdown/templates/visc_report/resources/template.tex:132]: Added in.....thanks for catching this

**Review by monicagerber — APPROVED** [review]
Looks great! 

Good to know about the pandoc latex template too.
(https://github.com/jgm/pandoc/blob/master/data/templates/default.latex)

I am going to add a bullet to the NEWS.md file, merge to master, then merge back to develop.

**Pull request merged**